content/html/concepts/tags/terms/picture/picture.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '<picture>'
+Description: 'The <picture> tag gives developers more flexibility when specifying image resources.'
```

Comment:
For term entries, meta descriptions should start with a verb. You can write something like: 
```suggestion
Description: 'Gives developers more flexibility when specifying image resources.'
```